src/components/menu-bar/menu-bar.jsx

```diff
@@ -298,39 +298,39 @@ class MenuBar extends React.Component {
                                     >
                                         {newProjectMessage}
                                     </MenuItem>
-                                ) : (
+                                ) : (this.props.showComingSoon ? (
                                     <MenuItemTooltip
                                         id="new"
                                         isRtl={this.props.isRtl}
                                     >
                                         <MenuItem>{newProjectMessage}</MenuItem>
                                     </MenuItemTooltip>
-                                )}
+                                ) : [])}
```

Comment:
It doesn't matter too much, but I think we're pretty consistent in this repo in using `prop ? () : null` for conditional rendering.  As an additional aside, I think it might be clearer if we started using `prop && ()` for this, to avoid confusion with nested ternaries.

Reply:
So, there's actually a problem in our code that makes using `null` this way unsafe!

In `menu.jsx`'s `MenuSection`, we're doing:

```
const addDividerClassToFirstChild = (child, id) => (
    React.cloneElement(child, {
        className: classNames(child.className, {
            [styles.menuSection]: id === 0
        }),
        key: id
    })
);

const MenuSection = ({children}) => (
    <React.Fragment>{
        React.Children.map(children, addDividerClassToFirstChild)
    }</React.Fragment>
);
```

It looks like React sees a `null` value as a child, and so we're attempting to get `(null)className`, throwing an error: 
```
Uncaught TypeError: Cannot read property 'className' of null
```

Reply:
Sounds like more reason to use `prop && ()`!